--- a/drivers/macintosh/apple-bce/apple_bce.c	2020-11-15 10:28:57.421385900 -0600
+++ b/drivers/macintosh/apple-bce/apple_bce.c	2020-11-15 10:28:44.246202500 -0600
@@ -432,5 +432,7 @@
 MODULE_AUTHOR("MrARM");
 MODULE_DESCRIPTION("Apple BCE Driver");
 MODULE_VERSION("0.01");
+// ROLANDO ZAPPACOSTA: added this for it to get autoloaded when built as a module:
+MODULE_ALIAS("pci:v0000106Bd00001801sv*sd*bc*sc*i*");
 module_init(apple_bce_module_init);
-module_exit(apple_bce_module_exit);
\ No newline at end of file
+module_exit(apple_bce_module_exit);
